lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
+    '`airbnb-prop-types` was given to a new maintainer and renamed',
+    packageRules: [
+      {
+        matchCurrentVersion: '^2',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['airbnb-prop-types'],
+        replacementName: 'prop-types-tools',
+        replacementVersion: '^2.17.0',
```

Comment:
The [Renovate developer docs, replacement presets](https://github.com/renovatebot/renovate/blob/main/docs/development/creating-editing-renovate-presets.md#replacement-presets) says:

> 3. Replacements should update the user to the first recommended version of the new dependency and not include any new changes - whether breaking or not - if they can be avoided

I assume that `2.17.0` is the first recommended version? Currently the `replacementVersion` uses the range syntax `^2.17.0`. The way I'm reading the rule means we should _pin_ the version instead:

```suggestion
        replacementVersion: '2.17.0',
```

Reply:
yes, that's the first one, but the semantics of a range are what users will want

Reply:
I'll let a maintainer respond to your point. :wink: